python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -259,12 +263,11 @@ def ggml_convert_qtype(tensor: torch.Tensor, qtype: int,
 
 
 def ggml_q_format_convet_cpu2xpu(tensor: torch.Tensor, num_elem: int, qtype: int):
-
-    invalidInputError(tensor.dtype == torch.uint8,
-                      "Input tensor must be uint8")
+    invalidInputError(tensor.dtype == torch.bfloat16,
```

Comment:
Will this change impact other features?

Reply:
NF4 applications e.g. QLoRA (zero2) will not be influenced. While maybe better add judgement `qtype == NF4`?